.github/workflows/docker.yml

```diff
@@ -34,11 +34,20 @@ jobs:
           username: ${{ secrets.DOCKERHUB_USERNAME }}
           password: ${{ secrets.DOCKERHUB_TOKEN }}
 
+      - name: Log in to GHCR
+        uses: docker/login-action@v2
+        with:
+          registry: ghcr.io
+          username: ${{ github.repository_owner }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
       - name: Calculate docker image tag
         id: set-tag
         uses: docker/metadata-action@master
         with:
-          images: matrixdotorg/synapse
+          images: |
+            matrixdotorg/synapse
+            ghcr.io/matrix-org/synapse
```

Comment:
How do we test this since we only run this on `v*` tags /  `develop` / `master` / `main`? Hope it works and ensure that when it hits `develop` it does build it?

Reply:
That looks like a plan to me.

Reply:
yup. Not sure there's anything better we can do. But since it's `develop` I'm not too worried. I did base this off an existing PR which was known to work about a year ago.